content/blog/lifehacking/2017-12-01_freedom_and_productivity.md

```diff
@@ -0,0 +1,199 @@
+of needs](https://en.wikipedia.org/wiki/Maslow's_hierarchy_of_needs),
+productivity would probably belong to the top level (self-actualization), since
+it assumes all other issues in your life have been addressed first. If we put
+aside the working life aspect, a great amount of knowledge can also apply to
+one's personal life and free time. Yet, it feels it's a sign that "success"
+compounds, to some extent: the less you have to worry about universal basic
+needs, the more you can have time to read, learn, get better and enhance
+yourself, which in turn makes you more suited to reach whatever goal you aim
+for. It doesn't only happen with productivity, but also with education: kids
+from richer families tend to perform better than kids from poorer families, etc
+(there are studies I don't have links to that try to prove this affirmation).
+
+## To seek the whys
+
+There's always something else that's tickling me about those posts: they seem
+to confound the means and the ends. Being productive isn't an end per se, it is
+a tool. And it seems that people forget what is their own motivation in the
+first place, what is their *why*. Why do you want to be more productive or
+successful anyways? Do you want to work less? Or to spend more time with your
+friends and family? Or to spend more time on hobbies and passions that can make
+you feel more fulfilled? Or is your motivation entirely different, and you want
+to raise up in your company's hierarchy, by rising up your throughput, the
+quantity or quality of your work? All motives are valable here of course. But
+there should be one in the first place, otherwise motivation might not quite
+follow, and thus the practice of productivity routines will be aborted very
+soon after they get started. This is often a very strong sign for myself, when
+I consider doing something, add it to a todo list and months later it is still
+there; maybe it wasn't that useful or valuable, and I can erase it with a light
+heart.
+
+And when the initial, kickstarting motivation is right there, the *why* should
+also help set a limit to the endless quest of productivity. To know when to
+stop. For instance, when you managed to get more time for a dedicated hobby,
+and you're plain satifised, what would be the point of squeezing even more in
+your limited free time? Remember, productivity is a means to an end, not an end
+by itself. Without this in mind, you could be chasing an endless goal.
+
+People get easily fooled into thinking that more working hours imply more
+quality work. That's a societal feeling: in a capitalistic society, being
+productive means contributing to growth thus the good economic health of a
+entity greater than yourself (your country, your region, the world). And since
+people mix up being productive with working long hours, the confusion is easy
+to make. But I think productivity is the opposite of this: trying to work the
+least number of hours to advance work on the most important and valuable
```

Comment:
I feel like this could be your title, or your subtitle (very click-baity, seemingly counter-intuitive, but the most valuable point you make to me).

Reply:
Funny because now that I re-read the sentence, out of context, it reads like productivity is the opposite of trying to work the least number of hours. I guess I would rephrase slightly, maybe like so: 

```
But I think productivity is the opposite of this. I think it is trying to work the ...
```